website/pages/en/new-chain-integration.mdx

```diff
@@ -1,75 +1,150 @@
 
----
+![Base block vs Extended block](/img/extended-vs-base-substreams-blocks.png)
+
+> NOTE: This improvement upon the Firehose requires chains make use of the EVM engine `geth version 1.13.0` and up.
+
+## EVM considerations - Difference between JSON-RPC & Firehose
 
-## Integrating a new Firehose-enabled chain
+While the JSON-RPC and Firehose are both suitable for subgraphs, a Firehose is always required for developers wanting to build with [Substreams](https://substreams.streamingfast.io). Supporting Substreams allows developers to build [Substreams-powered subgraphs](https://thegraph.com/docs/en/cookbook/substreams-powered-subgraphs) for the new chain, and has the potential to improve the performance of your subgraphs. Additionally, Firehose — as a drop-in replacement for the JSON-RPC extraction layer of `graph-node` — reduces by 90% the number of RPC calls required for general indexing.
 
-Integrating a new chain is also possible using the Firehose approach. This is currently the best option for non-EVM chains and a requirement for substreams support. Additional documentation focuses on how Firehose works, adding Firehose support for a new chain and integrating it with Graph Node. Recommended docs for integrators:
+- All those `getLogs` call and roundtrips get replaced by a single stream arriving into the heart of `graph-node` and by running filters on the in-memory Firehose Block; a _single_ block for all subgraphs it processes.
 
-1. [General docs on Firehose](firehose/)
-2. [Adding Firehose support for a new chain](https://firehose.streamingfast.io/integrate-new-chains/integration-overview)
-3. [Integrating Graph Node with a new chain via Firehose](https://github.com/graphprotocol/graph-node/blob/master/docs/implementation/add-chain.md)
+> NOTE: A Firehose-based integration for EVM chains will still require Indexers to run the chain's archive RPC node to properly index subgraphs. This is due to the Firehose's inability to provide smart contract state typically accessible by the `eth_call` RPC method. (It's worth reminding that `eth_calls` are not a good practice for developers)
+
+## Graph Node Configuration
+
+Configuring Graph Node is as easy as preparing your local environment. Once your local environment is set, you can test the integration by locally deploying a subgraph.
+
+1. [Clone Graph Node](https://github.com/graphprotocol/graph-node)
+2. Modify [this line](https://github.com/graphprotocol/graph-node/blob/master/docker/docker-compose.yml#L22) to include the new network name and the EVM JSON RPC compliant URL
+
+   > Do not change the env var name itself. It must remain ethereum even if the network name is different.
+
+3. Run an IPFS node or use the one used by The Graph: https://api.thegraph.com/ipfs/
+
+<details>
```

Comment:
Erm, I'm guessing this whole section written in HTML does not look like you expect. 😅

![CleanShot 2024-07-11 at 01 40 31@2x](https://github.com/graphprotocol/docs/assets/1059139/274cae7e-fa22-489e-baee-64c6b8b58576)

Does it really have to be an accordion? If so, I'm sure there's a way we can do it properly in MDX (probably by providing the GDS `Accordion` component), but for now it would probably be easier just to turn "Testing an EVM JSON-RPC by locally deploying a subgraph" into a heading, the `<ol>` into a Markdown numbered list, the `<code>`s into backtick-wrapped strings, etc. What do you think?

Reply:
@Giuliano-1 , it may be best to return to the idea of having this as a dedicated sub-page instead.

Reply:
The changes made for `lint` and `prettier` did not result well but it should work with the embedded HTML

Reply:
@benface this is what I see when I "git pull" this code (edited for `lint` and `prettier`) and test upload it to my personal github repo (or when navigating here: https://github.com/graphprotocol/docs/blob/giuliano/docrevision/website/pages/en/new-chain-integration.mdx)... it looks normal, can you give me an idea as to why it appears different on your end? 

The reason I want the `Accordion` to work is for a richer reading experience!

<img width="1070" alt="Screenshot 2024-07-22 at 11 07 48 PM" src="https://github.com/user-attachments/assets/9f22839d-a614-432d-858d-491558313b10">

Reply:
@Giuliano-1 – Looks like you're previewing the file in the GitHub UI, not in the actual Graph docs app. Do you have the repo cloned locally? If so, you can see what I see by running `pnpm dev` and going to `http://localhost:3000/en/new-chain-integration/` in your browser.

Reply:
(See the [`README`](https://github.com/graphprotocol/docs/blob/main/README.md) for more detailed instructions)